### PR TITLE
docs (metrics): update example metric

### PR DESCRIPTION
**Pull request by eloy-encord**

### CODE REVIEW

**Review by Gorkem-Encord — CHANGES_REQUESTED** [review]
In these example projects, both image-level and object-level writing is happening in the same iteration loop. We do not allow such thing and it may confuse the users, so we should note somewhere that 'you can either create an image-level metric or an object-level metric, not both at the same time'. We can also divide the iterating over data_units into two and comment one of them to not cause any confusion.

**Comment by frederik-encord** [comment]
What might want to do is having separate (sub)classes for image level and object level metrics and then enforce that they call the writer with the right parameters to avoid this confusion. Perhaps not in this PR though.

File: docs/docs/metrics/write-your-own.md [hunk 1]
```diff
@@ -24,74 +24,85 @@ You can use the following template to get started with writing your own metric.
...
 class ExampleMetric(Metric):
     def __init__(self):
         super().__init__(
             title="Example Title",
             short_description="Assigns same value and description to all objects.",
-            long_description=r"""For long descriptions, I can use Markdown to _format_ the text.
+            long_description=r"""For long descriptions, you can use Markdown to _format_ the text.
 
-I can, e.g., make a
+E.g. you can make a
```
Comment (frederik-encord) [inline docs/docs/metrics/write-your-own.md:43]:
```suggestion
For example, you can make a
```

File: src/encord_active/lib/metrics/example.py [hunk 2]
```diff
@@ -17,13 +17,13 @@ def __init__(self):
         super().__init__(
             title="Example Title",
             short_description="Assigns same value and description to all objects.",
-            long_description=r"""For long descriptions, I can use Markdown to _format_ the text.
+            long_description=r"""For long descriptions, you can use Markdown to _format_ the text.
 
-I can, e.g., make a
+E.g. you can make a
```
Comment (frederik-encord) [inline src/encord_active/lib/metrics/example.py:22]:
```suggestion
For example, you can make a
```

File: docs/docs/metrics/write-your-own.md [hunk 3]
```diff
@@ -24,74 +24,85 @@ You can use the following template to get started with writing your own metric.
 Your implementation should call `writer.write(<object_score>, <object>)` for every object in the iterator or use `writer.write(<frame_score>)` for every data unit in the iterator.
```
Comment (frederik-encord) [inline docs/docs/metrics/write-your-own.md:24]:
```suggestion
Your implementation should call `writer.write(<object_score>, <object>)` for every object in the iterator **OR** use `writer.write(<frame_score>)` for every data unit in the iterator.
```

Comment (frederik-encord) [reply docs/docs/metrics/write-your-own.md:24]: Perhaps add a line saying (for now) that one should not do both in the same metric definition.

**Review by frederik-encord — CHANGES_REQUESTED** [review]

**Review by frederik-encord — APPROVED** [review]

**Review by Gorkem-Encord — APPROVED** [review]

**Pull request merged**